README.md

```diff
@@ -2,13 +2,14 @@
 
 Genomics Status is a Tornado web app for visualizing information and statistics regarding SciLifeLab Genomics platform operations.
 
-Status interfaces with StatusDB; the CouchDB database instance we're using at SciLifeLab to store metadata in 
+Status interfaces with StatusDB; the CouchDB database instance we're using at SciLifeLab to store metadata in
 various forms. Document specifications for StatusDB are available in the internal wiki. Documentation about CouchDB can be found [here](http://guide.couchdb.org/).
 
 ## Installing and running genomics status
 
 **NOTE**: These steps assume that:
 * you're either running a python virtualenv or you do have root permissions.
+* you're running python version 2.X
```

Comment:
I think we should assume version 2.7. Woe to those who run python 2.6.

Reply:
Fine by me. I just noticed that python 3 was a nono.